scripts/scripts.js

```diff
@@ -137,6 +138,17 @@ async function loadLazy(doc) {
   loadCSS(`${window.hlx.codeBasePath}/styles/lazy-styles.css`);
   loadFonts();
 
+  function loadSocialShare() {
+  loadScript('https://buttons-config.sharethis.com/js/5cae6f10a8698b001266ffd9.js');
+  loadScript('https://platform-api.sharethis.com/js/sharethis.js#property=5cae6f10a8698b001266ffd9&product=inline-share-buttons');
+  loadScript('https://count-server.sharethis.com/v2.0/get_counts?cb=window.__sharethis__.cb&url=https%3A%2F%2Fwww.servicenow.com%2Fblogs%2F2023%2Fvancouver-release-genai-security-agility.html');
```

Comment:
the `url` parameter should not be hardcoded to one blog article. the way that it is in the PR right now, for every page it's going to look for https://www.servicenow.com/blogs/2023/vancouver-release-genai-security-agility.html